src/poliastro/twobody/orbit.py

```diff
@@ -580,6 +580,32 @@ def geostationary(
         altitude = geo_radius - attractor.R
         return cls.circular(attractor, altitude)
 
+    @classmethod
+    @u.quantity_input(a=u.m, ecc=u.one, w=u.rad / u.s, period=u.s)
+    def heliocentric(cls, attractor, a, ecc, w=None, period=None):
```

Comment:
Shouldn't this be called `heliosynchronous`? Heliocentric means a different thing.

Reply:
Sorry, a typo